router/network.go

```diff
@@ -182,11 +184,12 @@ func (network *NetHandleT) SendPost(ctx context.Context, structData integrations
 			//Detecting content type of the respBody
 			contentTypeHeader = http.DetectContentType(respBody)
 		}
+		mediaType, _, _ := mime.ParseMediaType(contentTypeHeader)
 
-		// If content type is not of type "*text*", overriding it with empty string
-		if !(strings.Contains(strings.ToLower(contentTypeHeader), "text") ||
-			strings.Contains(strings.ToLower(contentTypeHeader), "application/json") ||
-			strings.Contains(strings.ToLower(contentTypeHeader), "application/xml")) {
+		// If media type is not in some human readable format (text,json,xml), override the response with an empty string
+		// https://www.iana.org/assignments/media-types/media-types.xhtml
+		isHumanReadable, _ := regexp.MatchString(`^(text/[a-z0-9.-]+)|(application/([a-z0-9.-]+\+)?(json|xml))$`, mediaType)
+		if !isHumanReadable {
```

Comment:
@chandumlg this is my interpretation of why we are dropping the response body for some content types, please feel free to comment.

Reply:
The reason why we drop few metrics is because for some cases the response type might be in image format or something which is too heavy and might cause the status table to bloat up

Reply:
> The reason why we drop few metrics is because for some cases the response type might be in image format or something which is too heavy and might cause the status table to bloat up

Thanks @cisse21, do you believe the current comment is sufficient or should I change it?